src/Common/Commands.Common.Compute.Tests/Version2016-04-preview/ComputeManagementClientShould.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+namespace Microsoft.Azure.Commands.Common.Compute.Tests
+{
+    using Compute.Version2016_04_preview;
+    using Xunit;
+    using System.Linq;
+
+    namespace Version2016_04_preview
+    {
+        public class ComputeManagementClientShould
+        {
+            private IComputeManagementClient Client {get;} 
```

Comment:
These all need to be check-in tests.  Let's discuss what to do with this per the above.